cli/flag.go

```diff
@@ -17,16 +17,13 @@ limitations under the License.
 package cli
 
 import (
+	"fmt"
```

Comment:
Changes below has nothing to do with viper integration?  Please create a separate PR.